modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -157,6 +351,30 @@ content-type: text/html; charset=utf-8
 set-cookie: 1e2670d92730b515ce3a1bb65da45062=9b714eb87e93cf34853e87a92d6894be; path=/; HttpOnly; Secure; SameSite=None
 cache-control: private
 ----
++
+.. Verify that you can access each cluster application on port 443:
```

Comment:
Same comment as before regarding suggested sentence structure for these type of steps